### PR TITLE
flex support

### PR DESCRIPTION
**Pull request by 0xPxt**

### CODE REVIEW

**Review by chcmedeiros — COMMENTED** [review]
Just missing the bump in Makefile.version and new version snapshots

**Comment by 0xPxt** [comment]
Opened https://github.com/LedgerHQ/ledger-app-database/pull/254

**Review by chcmedeiros — APPROVED** [review]

File: tests_zemu/snapshots/fl-mainmenu/00001.png
Comment (emmanuelm41) [inline tests_zemu/snapshots/fl-mainmenu/00001.png:1]: This is not the latest zxlib version, as I don't see the double size line here. Any reason why we did not moved it to the latest version?

Comment (0xPxt) [reply tests_zemu/snapshots/fl-mainmenu/00001.png:1]: I think that was the latest zxlib version when this PR was opened, anyway, just opened a new PR to upgrade it : https://github.com/Zondax/ledger-alephzero/pull/29

Comment (emmanuelm41) [reply tests_zemu/snapshots/fl-mainmenu/00001.png:1]: I think I did a mistake and reviewed a really old PR hahaha forget about it!

**Pull request merged**